Draft/draft-ietf-rtcweb-stun-consent-freshness.xml

```diff
@@ -378,16 +391,25 @@
       an API hook that generates an event when consent has expired for a given
       5-tuple, meaning that transmission of data has ceased. This could
       indicate what application data is affected, such as media or data
-      channels.</t>
+      channels. The circuit breaker algorithm discussed in section 7.1 of
+      <xref target="I-D.ietf-rtcweb-rtp-usage"></xref> could also be one of
+      the reasons for ceasing transmission of media and to notify the
+      application about the loss of ability to transmit data.</t>
+
+      <t>WebRTC client application may in-turn notify WebRTC server about loss
+      of consent so that it can track frequency and duration of loss of
+      consent, e.g., to assist in troubleshooting application problems.</t>
```

Comment:
I know that David asked for these, but I think that this is bad.  If we intend to do this (and we should) it needs API support, not some text in this document.  All this does is beg the question: so how is this reported?